app/jobs/delete_user_job.rb

```diff
@@ -0,0 +1,14 @@
+class DeleteUserJob < ApplicationJob
+  queue_as :default
+  queue_with_priority PRIORITIES.fetch(:profile_deletion)
+  self.queue_adapter = :good_job
+
+  def perform(user:)
+    email = user.email
+    if user.destroy
+      Mailer.deletion_complete(email)
```

Comment:
What about explicit version`Mailer.deletion_complete(email).deliver_now` ?

Reply:
Or even `deliver_later` so problems with delivery will be restarted on its own without failing whole job (when using `deliver_now!` or be just ignored (when using `deliver_now`)?

Reply:
was missing the deliver_later, will add it